src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.html

```diff
@@ -30,6 +30,26 @@
       </div>
     </div>
 
+    <div class="row settings-section">
+      <div class="ev-md-container ev-panel-title" *ngIf="leaderboard.length >0 && selectedPhaseSplit != null">
```

Comment:
Minor: Please remove the unnecessary spacing here (`*ngIf="leaderboard.length >0 && selectedPhaseSplit != null"`) before and after the comparison operator

Reply:
@Sanji515 I think it would be more cleaner if we keep it something like ` *ngIf="leaderboard.length > 0 && selectedPhaseSplit != null"` instead of ` *ngIf="leaderboard.length>0 && selectedPhaseSplit!=null"`. what you say?

Reply:
Yeah but as we keep it without space in all the html files lets keep that way.